some changes

BloamBla opened the pull request:
- was changed a type of work with goals processing;
- was do a few small changes for optimization and code readability.

BloamBla committed:
some changes

- was changed a type of work with goals processing;
- was do a few small changes for optimization and code readability.

BloamBla committed:
translation fix

burymm commented on src/controllers/complete-to-do-template-controller.js:
```diff
@@ -12,7 +11,7 @@ angular.module('ToDo').controller('completeToDoTemplateController',
 
     $scope.translate = function () {
-        translationService.getTranslation($scope, myTranslate);
+        translationService.getTranslation($scope, selectLang);
```
I suppose selectedLang better name.

burymm commented on src/controllers/edit-done-or-delete-goals-controller.js:
```diff
@@ -0,0 +1,22 @@
+        };
+
+        $scope.translate = function () {
+            translationService.getTranslation($scope, selectLang);
```
selectedLang

burymm commented on src/controllers/save-to-loc-contr.js:
```diff
@@ -1,9 +1,8 @@
 angular.module('ToDo').controller('saveToLocContr',
-    function ($scope, $uibModalInstance, translationService, myTranslate, todosMemoryValue, MODAL_ANSWERS) {
+    function ($scope, $uibModalInstance, translationService, selectLang, todosMemoryValue, MODAL_ANSWERS) {
```
selectedLanguage

burymm commented on src/controllers/todo-template-controller.js:
```diff
@@ -1,5 +1,5 @@
 angular.module('ToDo').controller('todoTemplateController',
-    function (todo, $scope, $uibModalInstance, translationService, myTranslate) {
+    function (todo, $scope, $uibModalInstance, translationService, selectLang) {
```
+1

burymm commented on src/controllers/todo-template-controller.js:
```diff
@@ -12,7 +12,7 @@ angular.module('ToDo').controller('todoTemplateController',
 
         $scope.translate = function () {
-            translationService.getTranslation($scope, myTranslate);
+            translationService.getTranslation($scope, selectLang);
```
+1

burymm commented on src/index.js:
```diff
@@ -29,11 +30,4 @@ angular.module('ToDo').controller('todoController',
                 sessionStorage.setItem('currentLanguage', $scope.selectedLanguage);
```
why did you use session storage instead of localstorage?

BloamBla committed:
also some changes

- buttons replaced to icons;
- added possible to edit goal(before if you click to goal you could immediately edit goalб but now you need to click the edit button);
- fixed console errors.

burymm commented on src/controllers/edit-done-or-delete-goals-controller.js:
```diff
@@ -1,10 +1,13 @@
 
         $scope.save = function () {
-            $uibModalInstance.close(MODAL_ANSWERS.SAVE);
+            $scope.editIt();
```
what the meaning editIt?

BloamBla committed:
optimization

- made optimization to work with local storage.

BloamBla committed:
bug fix

- bug when you load from local storage nothin fixed.

BloamBla committed:
after review

- selectLang changed to selectedLanguage;
- now currentLanguage is stored in localstorage.
